docs/source/workflows/predictor_evaluation_workflows.rst

```diff
@@ -236,7 +236,7 @@ Then wait for the results to be ready:
     from citrine.jobs.waiting import wait_while_executing
 
     execution = workflow.executions.trigger(predictor.uid)
-    wait_while_executing(execution, print_status_info=True, collection = project.predictor_evaluation_executions)
+    (project.predictor_evaluation_executions, execution, print_status_info=True)
```

Comment:
i don't think you meant to make this change?

Reply:
The arguments are meant to be reordered, but you're right that I didn't mean to remove `wait_while_executing`